ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/PeerResult.java

```diff
@@ -49,6 +49,7 @@ static PeerResult fromEthPeer(final EthPeer peer) {
         .port(Quantity.create(peerInfo.getPort()))
         .id(peerInfo.getNodeId().toString())
```

Comment:
Did we also want to address the nodeId issue or is that being saved for another PR?

Reply:
I'm being pretty specific with this PR - i.e. do just enough to make cross Besu/GoQuorum work in PEEPS.
So while I think the NodeId is _probably_ worth doing - its something I'm happy to leave for now - its potentially a bigger issue, as its not just an API thing, it bleeds into the network model, and how we represent peers (which I don't want to investigate at this stage).